addons/core/translations/form/en-us.yaml

```diff
@@ -183,3 +183,5 @@ created:
 other_fields:
   label: Other Fields
   help: This information is provided by the server and cannot be edited in Boundary
+worker_filter:
+  label: Worker Filter
```

Comment:
Each letter of word is capitalized, tho im noticing that in some form fields only the first letter is capitalized. This pattern should probably be standardized.